examples/game_of_life/_helpers.py

```diff
@@ -26,9 +26,13 @@ def get_active_pin_map_id(self) -> str:
         Returns:
             The resource id of the pin map that is registered to the pin map service.
         """
-        return self._sequence_context.Execution.RunTimeVariables.GetValString(
-            "NI.MeasurementLink.PinMapId", 0x0
-        )
+        pin_map_id = "NI.MeasurementLink.PinMapId"
```

Comment:
> [bkeryan](https://github.com/bkeryan) [19 hours ago](https://github.com/ni/measurementlink-python/pull/719#discussion_r1595819279) • 
> I agree. Also rename to indicate that this is a run time variable name, not the actual id

`pin_map_id` sounds like the id itself. Call the variable or constant something like `pin_map_id_var` to show that it's the variable name.

Reply:
I have changed the variables names and made it a top level constant as well.